server/src/main/java/org/elasticsearch/cluster/metadata/ComposableIndexTemplate.java

```diff
@@ -267,6 +275,13 @@ public String getTimestampField() {
             this(in.readString());
         }
 
+        /**
+         * @return a mapping snippet for a backing index with `_timestamp` meta field mapper properly configured.
+         */
+        public Map<String, Object> getDataSteamMappingSnippet() {
```

Comment:
Noticed this typo as I was trying to pull up the method: `DataSteam` -> `DataStream`.